Oshanimi/SigninToScoring.feature

```diff
@@ -0,0 +1,146 @@
+	But user leaves the 'Password' text field empty
+	And user clicks the 'Next' button
+	Then the 'Password' text field is marked red. 
+	And 'Enter a password' appears in the validation errors region
+
+Scenario Outline: 1.1_05 Sign in with invalid phone number
+	Given  user has 'Choose an option' dialog box displayed on the screen
+	And user clicks on 'Use another account' button
+	When user fills the account 'Email or phone' text field with invalid phone number format 
+	And user clicks the 'Next' button
+	Then the 'email textfield'is marked red. 
+	And 'Enter email or phone number' appears in the validation errors region
+		
+		Examples: 
+			| Invalid phone number format | type                             |
+			| +79836120001                | Not registered on google account |
+			| +798361200                  | Incomplete phone number          |
+
+	
+Scenario Outline: 1.1_06 Sign in with valid email account 
+	Given  user has 'Choose an option' dialog box displayed on the screen
+	And user clicks on 'Use another account' button
+	When user fills the account email textbox with value 'beatricek006@gmail.com' 
+	And user clicks 'Next' button
+	And user fills the 'Password' text field with 'Monkeytail2@' 
+    And user clicks 'Next' button
+	Then user enters the scoring homepage
+
+Scenario Outline: 1.1_07 Sign in with valid phone number 
+	Given  user has 'Choose an option' dialog box displayed on the screen
+	And user clicks on 'Use another account' button
+	When user fills the account 'Email or phone number' textbox with value '+78972028989' 
+	And user clicks 'Next' button
+	And user fills the 'Password' text field with 'Monkeytail2@' 
+	And user clicks 'Next'button
+    Then user enters the scoring homepage
+
+Scenario Outline: 1.1_08 Sign in with valid saved Google account in 'Examples'
+	Given user has 'Choose an option' dialog box displayed on the screen
+	When multiple emails are displayed on the signin page to 'Choose an account' 
+	Then  user enters the scoring homepage through the states in the table
+	| Listed email                            | Action | State                                          |
+	| beatricek006@gmail.com                  | Clicks | Enter homepage                                 |
+	| oshanimioluwakemi@gmail.com             | Clicks | Enter homepage                                 |
+	| oluwakemisola.oshanimi@quantori.academy | Clicks | Enter homepage                                 |
+	| Use another account                     | Clicks | Redirects to sign in with valid google account |
+   And User clicks on 'Use another account'
+   And following features are displayed on the screen
+   
+	| Feature                  | Action							| Button | 
+	| Enter                    | Valid input					| Click  |
+	| Email or phone textfield | beatriceoluwakemisola@gmail.com| Next   |
+	| Enter your password      | VirusDetectedK006				| Next   | 
```

Comment:
Please, format it 
![image](https://user-images.githubusercontent.com/94701044/171501061-ff59ad27-a38f-4ed2-98fe-be565216a627.png)